ibis/pandas/execution/generic.py

```diff
@@ -461,6 +461,9 @@ def execute_count_distinct_series_groupby(
 
 @execute_node.register(ops.Arbitrary, SeriesGroupBy, type(None))
 def execute_arbitrary_series_groupby(op, data, _, aggcontext=None, **kwargs):
+    if op.how is None:
+        op.how = 'first'
```

Comment:
Do not mutate the `op` here. Assign `op.how` to a variable and operate with that.

Reply:
sounds good :+1: